tripal_analysis_expression/includes/TripalFields/local__expression_features/local__expression_features_formatter.inc

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @class
+ * Purpose:
+ *
+ * Display:
+ * Configuration:
+ */
+class local__expression_features_formatter extends ChadoFieldFormatter
+{
+
+  // The default label for this field.
+  public static $default_label = 'Expression Features';
+
+  // The list of field types for which this formatter is appropriate.
+  public static $field_types = array('local__expression_features');
+
+  // The list of default settings for this formatter.
+  public static $default_settings = array(
+    'setting1' => 'default_value',
+  );
+
+  /**
+   * @see ChadoFieldFormatter::settingsForm()
+   *
+   **/
+
+  public function settingsForm($view_mode, $form, &$form_state)
+  {
+
+  }
+
+  /**
+   * @see ChadoFieldFormatter::View()
+   *
+   **/
+
+  public function view(&$element, $entity_type, $entity, $langcode, $items, $display)
+  {
+    $rows = $items[0]['value'];
+    if ($rows == []) {
+      $rows = 'No features exist.';
```

Comment:
This shouldn't be a string since the theme('table') function expects an array. It would be better if you simply add `$element[0] => ['#markup' => 'No feature found'];` and return if there are no rows. That way you don't need to use theme('table') if there are no features.